### ISSUE TITLE
clamp causes closed path frames not meet

### ISSUE THREAD

**Issue by vorg** [dmnsgn/frenet-serret-frames#10]
Do we know why clamp is here? https://github.com/dmnsgn/frenet-serret-frames/blob/main/index.js#L110

```js
let theta = Math.acos(clamp(vec3.dot(firstNormal, lastNormal), 0, 1));
theta /= frames.length - 1;
```
![Screenshot 2023-07-10 at 10 59 01](https://github.com/dmnsgn/frenet-serret-frames/assets/171001/26a1ddf0-19dc-4c87-9c8d-9a6eada9bb1a)

Removing clamp fixes the issue.

**Comment by dmnsgn** [comment]
It should actually be clamp between -1 and 1. Will fix in the next version.

**Comment by dmnsgn** [comment]
Fixed in [v3.0.0](https://github.com/dmnsgn/frenet-serret-frames/releases/tag/v3.0.0)
